api/app/signals/apps/signals/models/routing_expression.py

```diff
@@ -14,6 +17,7 @@ class RoutingExpression(models.Model):
         related_name='routing_department'
     )
     _department = models.ForeignKey(Department, on_delete=models.CASCADE)
+    employee = models.ForeignKey(to=User, on_delete=models.SET_NULL, blank=True, null=True)
```

Comment:
SIA/Signalen has access control based on general Django groups (called roles in SIA/Signalen context) and department and signal category.

By allowing this one rule to assign both a user (e.g. with a relation to departments X, Y and Z) and a department W it is possible to make a "incorrect" user assignment. That is, it is possible to assign a user to a Signal who is not allowed to see that Signal.